andela/ah-kgl-avengers-frontend
# #165020165 creates login using google and facebook

### What does this PR do?
- login with facebook and google

### Description of Task to be completed?
- User should sign up or login using facebook and google
### How should this be manually tested?
- Clone the repository locally with `git clone https://github.com/andela/ah-kgl-avengers-frontend.git` command.
- Check out to this branch by running `git checkout  ft-social-login-via-fg-google`.
- Install dependencies by running `npm install`.
- Update the .env file according to the .env.sample file
- Start the server with `npm start` and put this url `http://localhost:8000/signup`
- sign up or log in with your favorite social media
### Any background context you want to provide?
- Please make sure your social media has an email no telephone from social media
### What are the relevant pivotal tracker stories?
[#165020165 Users should be able to Login via Google Facebook](https://www.pivotaltracker.com/story/show/165020165)
### Screenshots (if appropriate)
- none
### Questions:
